docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -145,7 +145,7 @@ The referenced Secret must contain the following connection information:
 - `url`: the URL to reach the Elasticsearch cluster
 - `username`: the username of the user to be authenticated to the Elasticsearch cluster
 - `password`: the password of the user to be authenticated to the Elasticsearch cluster
-- `ca.crt`: the CA certificate in PEM format to secure communication to the Elasticsearch cluster (optional)
+- `ca.crt`: the contents of the CA certificate in PEM format to secure communication to the Elasticsearch cluster (optional)
```

Comment:
I'm not against this change, but I don't get what suggests that the path is expected and not the certificate itself when we write "the CA certificate in PEM format"?

When a path is expected, it is stated like that:
```
> grep 'Path to' -rh docs
|ca-dir |"" |Path to a directory containing a CA certificate (tls.crt) and its associated private key (tls.key) to be used for all managed resources. Effectively disables the CA rotation and validity options.
|config |"" | Path to a file containing the operator configuration.
|webhook-cert-dir |"{TempDir}/k8s-webhook-server/serving-certs" |Path to the directory that contains the webhook server key and certificate.
|webhook-cert-dir     | /tmp/k8s-webhook-server/serving-certs | Path to mount the certificate.
```

What bothers me a little is that this would mean that we have to review all the same occurrences in the doc which would lead to the same confusion?
```
# docs/orchestrating-elastic-stack-applications/logstash.asciidoc:
<4> The CA certificate in PEM format to secure communication to the Elasticsearch cluster (optional).

# docs/orchestrating-elastic-stack-applications/apm-server.asciidoc:
NOTE: The `elasticsearch-ca.crt` file must contain the CA certificate of the Elasticsearch cluster you want to use with the APM Server.

# docs/orchestrating-elastic-stack-applications/accessing-elastic-services.asciidoc:
- `ca.key`: The private key to the CA certificate.

# docs/orchestrating-elastic-stack-applications/elasticsearch/remote-clusters.asciidoc:
. Create a config map with the CA certificate you just extracted:

# docs/orchestrating-elastic-stack-applications/elasticsearch/transport-settings.asciidoc:
The CA certificate must be stored in the secret under `ca.crt` and the private key must be stored under `ca.key`.

1 docs/reference/api-docs.asciidoc:
Certificate is a reference to a Kubernetes secret that contains the CA certificate and private key for generating node certificates. The referenced secret should contain the following:

# docs/reference/api-docs.asciidoc:
- `ca.crt`: The CA certificate in PEM format. - `ca.key`: The private key for the CA certificate in PEM format.
```

Reply:
I'm happy to convert it to an internal KB article if you think this doesn't warrant it. I think for clarity, especially as some of our users are non-native speakers, it doesn't hurt to be extra thorough with the terms we use.

FWIW, I had to double check myself as I couldn't tell from the docs alone that it was what we'd expect.

Reply:
The rule is that if the word `path` is not written, it is not a `path` that is expected.

It sounds wrong to create a KB article as an alternative for this, so ok it doesn't hurt, let's add it, but let's try to stay consistent.

Caould you identify other references to `certificates` in the documentation where there is the same ambiguity?